lib/utils.js

```diff
@@ -25,12 +25,16 @@ function cleanFile(pathToFile) {
  */
 function getFlagArrayFromObject(obj) {
   return Object.keys(obj).reduce((arr, key) => {
-    const flag = `--${_kebabCase(key)}`;
-    if (typeof obj[key] === 'boolean') {
+    const flag = `--${key.toLowerCase()}`;
+    if (key === 'noUpdateNotifier' && typeof obj[key] === 'boolean') {
```

Comment:
Would you mind explaining this change?

Reply:
All the documented flags are lowercase no spaces/dashes except --ignored-file. noUpdateNotifier isn't documented in themekit but we use it to prevent the node-themekit users from getting a warning to update themekit, and it is still a kebab case flag so separate case for that.